test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -69,6 +69,18 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.Equal(doc.Id, id);
         }
 
+        protected async Task NotificationHubTest()
+        {
+            // NotificationHub tests need the following environment vars:
+            // "AzureWebJobsNotificationHubsConnectionString" -- the connection string for NotificationHubs
```

Comment:
Please send a not to team on setting this up correctly when this gets in, so we can update our machines to run the tests successfully.